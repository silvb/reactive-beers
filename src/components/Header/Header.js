import React from 'react'

const Header = () => (
  <header>
    <h1>Reactive Beers 🍺🍺🍺</h1>
  </header>
)

export default Header