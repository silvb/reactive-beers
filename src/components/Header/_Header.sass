header
  height: 20vh
  background: #4f3a4b
  width: 100%
  margin-bottom: 20px
  display: flex
  align-items: center
  justify-content: center
  box-shadow: 0 4px 4px rgba(black, 0.4)

  h1
    color: #e75152
    text-shadow: 2px 2px 0 rgba(black, 0.2)
    font-size: 64px
    letter-spacing: 2px
    font-family: 'Lobster', cursive
    font-weight: bold
    text-align: center
    vertical-align: middle
