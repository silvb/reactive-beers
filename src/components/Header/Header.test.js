import React from 'react'
import renderer from 'react-test-renderer'
import Header from './Header'

describe('Header', () => {
  it('renders without crashing', () => {
    const component = renderer.create(
      <Header />
    )

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})