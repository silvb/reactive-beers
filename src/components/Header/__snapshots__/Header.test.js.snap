// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<header>
  <h1>
    Reactive Beers 🍺🍺🍺
  </h1>
</header>
`;
